tests/rest/test_single_topology_rest.py

```diff
@@ -99,20 +108,22 @@ def test_single_topology_rest_vacuum(mol_pair, temperature_scale_interpolation_f
         assert energy_scale < 1.0
 
         if has_rotatable_bonds or has_aliphatic_rings:
-            assert 0 < len(st_rest.target_propers)
+            assert 0 < len(st_rest.candidate_propers)
 
             if energy_scale < 1.0:
                 assert U_proper < U_proper_ref
 
-        def get_proper_subset_energy(state: GuestSystem, ixn_idxs):
+        def compute_proper_energy(state: GuestSystem, ixn_idxs):
```

Comment:
typing on `ixn_idxs: list/NDArray[int]`?

Reply:
Added in https://github.com/proteneer/timemachine/pull/1524/commits/97814ada93445aceb8d4fef4b51aa5f6951e5274